production

```diff
@@ -0,0 +1,43 @@
+# Dockerfile # Mapknitter
+# https://github.com/publiclab/mapknitter/
+# This image deploys Mapknitter!
+
+FROM ruby:2.4.6-stretch
+
+# Set correct environment variables.
+ENV HOME /root
+
+# Backported GDAL
+RUN echo "deb http://packages.laboratoriopublico.org/publiclab/ stretch main" > /etc/apt/sources.list.d/publiclab.list
+
+# We bring our own key to verify our packages
+COPY sysadmin.publiclab.key /app/sysadmin.publiclab.key
+RUN apt-key add /app/sysadmin.publiclab.key
+
+# Install dependencies for Mapknitter
+RUN apt-get update -qq && apt-get install -y \
+  nodejs gdal-bin curl procps git imagemagick python-gdal zip
+
```

Comment:
@icarito Anything else we could remove besides the chromedriver??

Reply:
We don't need git during production either. Also while we are optimizing we might as well drop apt caches (`apt-cache clean` after installation).